atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -136,16 +137,25 @@ def run(self):
                 registry_image.registry = registry
                 if self.need_skopeo_push():
                     self.push_with_skopeo(registry_image, insecure, docker_push_secret)
+                    digests = get_manifest_digests(registry_image, registry,
+                                                   insecure, docker_push_secret)
```

Comment:
Does it make sense to retry for skopeo push also?

Reply:
yeah as we don't know if skopeo will fail also, we can include it in retry